azure/scope/managedcontrolplane.go

```diff
@@ -418,7 +434,7 @@ func (s *ManagedControlPlaneScope) APIServerLBName() string {
 }
 
 // APIServerLBPoolName returns the API Server LB backend pool name.
-func (s *ManagedControlPlaneScope) APIServerLBPoolName(_ string) string {
```

Comment:
what was going on here earlier?

Reply:
that was just a cleanup leftover from https://github.com/kubernetes-sigs/cluster-api-provider-azure/commit/cb9494062a4076368013ebcdfe85726eddf44a7a